src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +49,15 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
```

Comment:
The `OrderService` is instantiated but not used in the code. If it's not needed, consider removing it. Otherwise, ensure it is used appropriately to fulfill its intended purpose.